### ISSUE TITLE
Router objects

### ISSUE THREAD

**Issue by elGusto** [nicolaslopezj/meteor-router-layer#3]
Hi!
Super cool initiative to make this package.

But I think there is a problem with the design of this package:

``` javascript
/**
 * Check if uses iron router
 */
if (_.has(Package, 'iron:router')) {
  RouterLayer.router = 'iron-router';
  RouterLayer.ironRouter = Package['iron:router'].Router;
}

/**
 * Check if uses flow router
 */
if (_.has(Package, 'kadira:flow-router')) {
  RouterLayer.router = 'flow-router';
  if (!_.has(Package, 'kadira:blaze-layout')) {
    throw new Meteor.Error('router-layer', 'If you use kadira:flow-router you must add kadira:blaze-layout');
  }
  RouterLayer.flowRouter = Package['kadira:flow-router'].FlowRouter;
  RouterLayer.blazeLayout = Package['kadira:blaze-layout'].BlazeLayout;
}
```

I think what this does is create some copy of the FlowRouter object that is not the one used in the main application. I'm still new to JS so I may be mistaking.

In my opinion this code is not necessary, you are able to call FlowRouter directly in the flowrouter implementation file (and same thing for IR). I tried it and it works, going to submit a PR in a few moments unless you have a specific issue for the choice of the above code.

Exemple:
(edited: I didn't read the watchPathChange doc, it has to be in a tracker function in order to be reactive, like a template helper) 

``` javascript
RouterLayer._getPath = function() {
    return FlowRouter.current().path;
  }
```

Let me know of your thoughts!

**Comment by nicolaslopezj** [comment]
In fact that would be the correct way to do it. But for some strange reason the `FlowRouter` and `Router` (for iron router) variables where not available in the global scope, only using the `Package[]` way.

Maybe its because the weak dependency

``` js
api.use(['iron:router@1.0.9', 'kadira:flow-router@2.1.1', 'kadira:blaze-layout@2.0.0'], { weak: true });
```

**Comment by elGusto** [comment]
Hey did you have time to take a look at the pattern in the PR I submitted?
I tested it with a refactor of the meteoric package to make it router agnostic and it works quite well.

I had problems of routes not being found when using the original package because of the two different instances issue.

I think this package can be very useful for the community, let's not leave it to rot in the atmosphere void.
